docs/runtimes/mlrun_code_annotations.ipynb

```diff
@@ -0,0 +1,1253 @@
+{
```

Comment:
<ul><li>Make sure that every <code># mlrun: start-code</code>&nbsp;has a corresponding <code># mlrun: end-code</code>&nbsp; especially for nested annotations.</li><li>Only 1 MLRun function can have a nameless annotation per notebook.</li><li>Do not use multiple&nbsp;<code># mlrun: start-code</code>&nbsp;nor multiple&nbsp;<code># mlrun: end-code</code>&nbsp;annotations in a single code cell. Only the 1st appearance of each will be used.</li><li>Using single annotations:</li></ul><ol><li class="ql-indent-1">Use a&nbsp;<code># mlrun: start-code</code>&nbsp;alone, and it will include code blocks from the annotation to the end of the notebook.</li><li class="ql-indent-1">Use a <code># mlrun: end-code</code>&nbsp;alone, and it will include code blocks from the beginning of the notebook to the annotation.</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/1456/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>